src/encoding/encoder.rs

```diff
@@ -47,13 +47,13 @@ impl Encoder {
                 self.output.extend_from_slice(length.as_bytes());
                 self.output.push(b':');
                 self.output.extend_from_slice(s);
-            },
+            }
```

Comment:
I was't using nightly rustfmt, so not all of configuration options in `rustfmt.toml` were picked up. I reformatted with nightly rustfmt, and the diffs went away. I just force-pushed a new version of the diff without those diffs, to avoid creating formatting diffs in one commit, and immediately undoing them in the next.